mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function will generate random floating numbers between numbers 1 - 10.
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+//the concat() function joins or combines two or more strings.
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
Great job!
You've only learnt string concatenation and interpolation methods last week but I'm impressed that you discovered the built in concat function.

Reply:
Thank you Biruk! You have been a great mentor to me.

Reply:
It has been great working with you, Chioma!!
Thanks!